controllers/contacts/createContact.js

```diff
@@ -0,0 +1,29 @@
+import { createContact, getContactByName } from "#service/index.js";
+
+async function createContacts(req, res, next) {
+  const { name, email, phone, favorite } = req.body;
+  try {
+    const existingContact = await getContactByName(name);
```

Comment:
Jeżeli w tym miejscu masz tylko stworzyć kontakt to trzeba byłoby zwalidować pola tylko.  No chyba że zakładasz że nie może być kilku osób o tych samych imieniu? W normalnych warunkach mogą być możesz mieć dwóch kamilów :)

Reply:
Poprawione